README.md

```diff
@@ -40,6 +44,35 @@ a token provided by the web app.
 As a starting point please check ```ApiModule``` class - it provides an
 entry point you need to know to communicate with the web application. 
 
+### Troubleshooting
+
+#### Gradle synchronization failure
+
+If you encounter this error:
+
+```
+Gradle sync failed: Could not get unknown property 'keyAlias' for project ':petunia' of type org.gradle.api.Project.
+         Consult IDE log for more details (Help | Show Log)
+```
+
+Create a file named gradle.properties in the root directory of the project with the following content:
```

Comment:
gradle.properties.sample already exist in 'petunia' module so user should only renamed it to 'gradle.properties'.
Btw, this file should not be on root but in petunia module.